### PR TITLE
Convert user profile account standing page to TypeScript

### PR DESCRIPTION
**Pull request by nanaya**
- [x] depends on #8052

### CODE REVIEW

File: resources/assets/lib/profile-page/account-standing.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            </tbody>
+          </table>
+        </div>
+      </div>
+    );
+  }
+
+  private readonly renderColumn = (column: Column, history: UserAccountHistoryJson) => (
+    <td key={column} className={classWithModifiers(`${bn}__table-cell`, column)}>
+      {React.createElement(content[column], { history })}
```
Comment (notbakaneko) [inline resources/assets/lib/profile-page/account-standing.tsx:136]:
```
const Elem = content[column];
<Elem .... />
```
also works

**Review by notbakaneko — APPROVED** [review]
should be fine

File: resources/assets/lib/profile-page/account-standing.tsx (hunk 1 above)
Comment (nanaya) [reply resources/assets/lib/profile-page/account-standing.tsx:136]: Well if you see the last comment :upside_down_face:

Comment (notbakaneko) [reply resources/assets/lib/profile-page/account-standing.tsx:136]: 🙃

**Pull request merged**